src/main/java/core/model/dto/UserRequestDto.java

```diff
@@ -1,8 +1,23 @@
 package core.model.dto;
 
+import core.lib.FieldMatch;
+import core.lib.ValidateEmail;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@FieldMatch(message = "Passwords are different.",
+        field = "password",
+        fieldMatch = "repeatPassword")
 public class UserRequestDto {
+    @ValidateEmail
+    @Size(min = 4)
     private String email;
+    @NotNull(message = "Please, don't leave this field empty.")
```

Comment:
```suggestion
```
we check this field on null value in validator class

Reply:
IDK on your code in the validator class, but in my case there are 2 cases following scenario if password != we compare password and repeated password in other case (this means password == null) we check if repeated password is also null (and if it is it means password == repeated password == null)